suite-native/module-dev-utils/src/components/ProductionDevInfo.tsx

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
 
-import { Card, VStack } from '@suite-native/atoms';
+import { Button, Card, VStack } from "@suite-native/atoms";
 
 import { CopyLogsButton } from './CopyLogsButton';
 import { BuildInfo } from './BuildInfo';
+import { clearStorage } from "@suite-native/storage";
 
 export const ProductionDevInfo = () => (
     <Card>
         <VStack spacing="medium">
+            <Button onPress={clearStorage}>Clear storage</Button>
```

Comment:
Since it's production version we should be more careful, I would suggest something like `Wipe all app data (dangerous!)` because in this part could be assible by regular user.

Reply:
Do you think 'Wipe all app data' and using Danger button is enough?